### ISSUE TITLE
Vulnerability in pycrypto 2.6.1 of OWASP/django-DefectDojo

### ISSUE THREAD

**Issue by curphey** [OWASP/django-DefectDojo#266]
Your **pycrypto 2.6.1** library, a direct dependency in your **OWASP/django-DefectDojo** repository, is subject to vulnerability [SID-3175: Arbitrary Remote Code Execution via Buffer Overflow](https://www.sourceclear.com/registry/vulnerabilities/3175).

A direct dependency is a library that is specified in your project’s setup.py.  A vulnerability in a direct dependency can be fixed by updating the version of the library in your project and rebuilding it.

This issue has not yet been fixed in a released version of the library. Please use the steps located at the [SourceClear vulnerability page](https://www.sourceclear.com/registry/vulnerabilities/3175).

Be sure to build, test, and re-scan your project to ensure this fix has been successful.

**Comment by devGregA** [comment]
Hi @curphey, thank you for the reports. We will investigate what you have reported, but I am closing the issues due to the the self promotion.

**Comment by curphey** [comment]
One of the most crazy reasons I have ever heard to close a security bug. Not sure what else to say.

**Comment by mtesauro** [comment]
I took the time to look at all the Source Clear issues entered against Dojo back on April 6th- full details are in this [Google Doc](https://docs.google.com/document/d/1Z0lJPIR1x2D0Q2_XqfYh-ZPvuK8q3gcQNxbTtgzRN38/edit?usp=sharing).  I figured I should actually add the results to the issues so any of our users can see that we take these things seriously.

Overall for all the issues, I found:
- 3 False Positives
- 1 Duplicate
- 3 Already planned updates of Python modules which we're either not vulnerable or maybe in certain edge cases.

For this specific issue, the TLDR is: False positive.  What we should do about this: Switch to Python Cryptography (more actively maintained) and use fernet: https://cryptography.io/en/latest/fernet/ when we get the chance.  Right now, we’re not vulnerable to this.
